### PR TITLE
Update directory.txt

### PR DESCRIPTION
**Pull request by madskjeldgaard**
Add a couple of my quarks

### CODE REVIEW

**Comment by dyfer** [comment]
Hey @madskjeldgaard I think you have a typo in the first address :)

Also, a _really_ minor thing (not worth opening a PR in the other repo) - in the `PolyBuf.quark` you have ext_dependancy instead of ext_depend**e**ncy - which was a typo in SC help for a long time...

**Comment by madskjeldgaard** [comment]
should be good now @dyfer !

File: directory.txt [hunk 1]
```diff
@@ -1,6 +1,7 @@
 3Dj=https://github.com/supercollider-quarks/3Dj
 adclib=https://github.com/supercollider-quarks/adclib
 AllGui=https://github.com/supercollider-quarks/AllGui
+AllBuf=https://github.com/polybuf/madskjeldgaard/allbuf
```
Comment (dyfer) [inline directory.txt:4]: This is still not a valid URL :)

**Review by dyfer — CHANGES_REQUESTED** [review]

**Comment by madskjeldgaard** [comment]
Haha, very sorry! Fixing it now

On 05-08-2021, Marcin Pączkowski wrote:
***@***.*** requested changes on this pull request.

**Pull request merged**